homeassistant/components/sensor/discogs.py

```diff
@@ -25,47 +27,81 @@
 
 DEFAULT_NAME = 'Discogs'
 
-ICON = 'mdi:album'
-
-SCAN_INTERVAL = timedelta(hours=2)
+ICON_RECORD = 'mdi:album'
+ICON_PLAYER = 'mdi:record-player'
+UNIT_RECORDS = 'records'
+
+SCAN_INTERVAL = timedelta(minutes=10)
+
+SENSORS = {
+    'collection': {
+        'name': 'Collection',
+        'icon': 'mdi:album',
+        'unit_of_measurement': 'records'
+    },
+    'wantlist': {
+        'name': 'Wantlist',
+        'icon': 'mdi:album',
+        'unit_of_measurement': 'records'
+    },
+    'random_record': {
+        'name': 'Random Record',
+        'icon': 'mdi:record_player',
+        'unit_of_measurement': None
+    },
+}
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_TOKEN): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=SENSORS):
```

Comment:
Copy the `SENSORS` dict to a list.

Reply:
Are you refering to what I had to undo before? https://github.com/home-assistant/home-assistant/pull/19443/commits/56ab36492264a33829ca7884ea08b5820d434170

Reply:
The conditions need to be a list so do:
```py
default=list(SENSORS)
```
I wrote that in my earlier comment too.